lib/pages/welcome_page.dart

```diff
@@ -59,6 +59,63 @@ class WelcomePage extends StatelessComponent {
       ])
     ]);
     yield Spacer(height: Unit.pixels(100));
+    yield h2(
+      [
+        text('Why join?'),
+      ],
+    );
+    yield div(classes: [
+      'activities-section'
+    ], [
+      div(classes: [
+        'activities-item'
+      ], [
+        img(classes: ['activities-icon'], src: '/images/4857010.png'),
+        h3([text('Meetings')]),
+        p(classes: [
+          'activities-text'
+        ], [
+          text('Monthly meetings to connect and share news and achievements')
+        ])
+      ]),
+      div(classes: [
+        'activities-item'
+      ], [
+        img(classes: ['activities-icon'], src: '/images/6491438.png'),
+        h3([text('Meet Flutter team')]),
+        p(classes: [
+          'activities-text'
+        ], [
+          text(
+              'A guest from the Flutter team come and meet with us occasionally to share news, tips and answer questions.')
+        ])
+      ]),
+      div(classes: [
+        'activities-item'
+      ], [
+        img(classes: ['activities-icon'], src: '/images/4892807.png'),
+        h3([text('Volunteering & Experiences')]),
+        p(classes: [
+          'activities-text'
+        ], [
+          text(
+              'Conference planning where Flutteristas volunteers with their talent and knowledge to produce an outstanding experience that reflect the member\'s core competent.')
```

Comment:
Suggest rewording - "Flutteristas volunteer their time and talents in producing outstanding Flutterista conference experiences that reflect our members' core competencies."